front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,356 @@
+
+    const formattedDepartureTime = getStopTime(departureTime);
+    const formattedArrivalTime = getStopTime(arrivalTime);
+
+    return { departureTime: formattedDepartureTime, arrivalTime: formattedArrivalTime };
+  }
+
+  return (
+    <Document>
+      <Page wrap={false} style={styles.page} size={[1344]}>
+        <View style={styles.alertBanner}>
+          <Image src={iconAlert} style={styles.alertIcon} />
+          <Text style={styles.simulationTitle}>{t('simulation')}</Text>
+          <Text style={styles.message}>{t('warningMessage')}</Text>
+        </View>
+        <View style={styles.numberDateBanner}>
+          <View style={styles.stdcmTitleBox}>
+            <View style={styles.stdcm}>
+              <Text style={styles.title}>{t('stdcm')}</Text>
+              <Text style={styles.creation}>{t('stdcmCreation')}</Text>
+            </View>
+          </View>
+          <View style={styles.numericInfo}>
+            <Text style={styles.number}>
+              n°
+              {number}
+            </Text>
+            <Text style={styles.creationDate}>{formattedDate}</Text>
+          </View>
+          <Image src={logoSNCF} style={styles.sncfLogo} />
+        </View>
+
+        <View style={styles.rcInfo}>
+          <View style={styles.rcBox}>
+            <Text style={styles.rcName}>{fakeInformation.rcName}</Text>
+            <Text style={styles.rcPersonName}>{fakeInformation.rcPersonName}</Text>
+          </View>
+          <View style={styles.rcBox}>
+            <Text style={styles.rcPhoneNumber}>{fakeInformation.rcPhoneNumber}</Text>
+            <Text style={styles.rcMail}>{fakeInformation.rcMail}</Text>
+          </View>
+          <View style={styles.rcBox}>
+            <View style={styles.stdcmApplication}>
+              <Text style={styles.applicationDate}>{t('applicationDate')}</Text>
+              <Text style={styles.date}>{formatDay()}</Text>
+              <Text style={styles.referencePath}>{t('referencePath')}</Text>
+              <Text style={styles.pathNumber}>{fakeInformation.path_number1}</Text>
+            </View>
+          </View>
+        </View>
+        <View style={styles.convoyAndRoute}>
+          <View style={styles.convoy}>
+            <Text style={styles.convoyTitle}> {t('convoy')}</Text>
+            <View style={styles.convoyInfo}>
+              <View style={styles.convoyInfoBox1}>
+                <Text style={styles.convoyInfoTitles}>{t('speedLimitByTag')}</Text>
+                <Text style={styles.convoyInfoData}>
+                  {stdcmData?.simulation.speed_limit_tags || '-'}
+                </Text>
+                <Text style={styles.convoyInfoTitles}>{t('towedMaterial')}</Text>
+                <Text style={styles.convoyInfoData}>-</Text>
+                <Text style={styles.convoyInfoTitles}>{t('maxSpeed')}</Text>
+                <Text style={styles.convoyInfoData}>
+                  {rollingStockData?.max_speed && rollingStockData?.max_speed * 3.6} km/h
```

Comment:
Remove all the ? and the test, and use a string like this:
```suggestion
                  {`${rollingStockData.max_speed * 3.6} km/h`}
```

Reply:
fixed